--- omniORB-4.1.7/src/tool/omniidl/cxx/cccp/cccp.c~	2006-08-02 16:26:34.000000000 +0200
+++ omniORB-4.1.7/src/tool/omniidl/cxx/cccp/cccp.c	2013-12-07 17:48:14.579353579 +0100
@@ -8034,7 +8034,7 @@
       else
 	parse_error = macarg (NULL_PTR, 0);
       if (parse_error) {
-	error_with_line (line_for_error (start_line), parse_error);
+	error_with_line (line_for_error (start_line), "%s", parse_error);
 	break;
       }
       i++;
