diff -urNp -x '*.orig' omniORB-4.2.2.org/include/omniORB4/sslContext.h omniORB-4.2.2/include/omniORB4/sslContext.h
--- omniORB-4.2.2.org/include/omniORB4/sslContext.h	2017-02-20 23:45:59.000000000 +0100
+++ omniORB-4.2.2/include/omniORB4/sslContext.h	2021-10-04 22:46:40.905231485 +0200
@@ -102,7 +102,7 @@ class sslContext {
   virtual ~sslContext();
 
  protected:
-  virtual SSL_METHOD* set_method(); 
+  virtual const SSL_METHOD* set_method(); 
   // Default to return SSLv23_method().
 
   virtual void set_supported_versions(); 
diff -urNp -x '*.orig' omniORB-4.2.2.org/src/lib/omniORB/orbcore/ssl/sslContext.cc omniORB-4.2.2/src/lib/omniORB/orbcore/ssl/sslContext.cc
--- omniORB-4.2.2.org/src/lib/omniORB/orbcore/ssl/sslContext.cc	2017-02-20 23:45:59.000000000 +0100
+++ omniORB-4.2.2/src/lib/omniORB/orbcore/ssl/sslContext.cc	2021-10-04 22:46:40.905231485 +0200
@@ -141,7 +141,7 @@ sslContext::PeerDetails::~PeerDetails()
 }
 
 /////////////////////////////////////////////////////////////////////////
-SSL_METHOD*
+const SSL_METHOD*
 sslContext::set_method() {
   return OMNI_CONST_CAST(SSL_METHOD*, SSLv23_method());
 }
